docs/buildah-commit.md

```diff
@@ -33,11 +33,14 @@ Control the format for the image manifest and configuration data.  Recognized
 formats include *oci* (OCI image-spec v1.0, the default) and *docker* (version
 2, using schema format 2 for the manifest).
 
+**--rm**
+Remove the container content after committing it to an image.  Default leaves the container content.
```

Comment:
I'd suggest:  "Remove the container and its content after committing it to an image.  Default leaves the container and its content in place.